src/api/routers/participants/participantsCreation.ts

```diff
@@ -50,19 +69,29 @@ export async function createParticipant(req: ParticipantRequest, res: Response)
   const types = await ParticipantType.query().findByIds(participantRequest.participantTypes);
   const apiRoles = await ApiRole.query().findByIds(participantRequest.apiRoles);
 
+  let site;
+  if (!participantRequest.siteId) {
+    // create site (UID2-2631)
```

Comment:
We can remove this comment now? Or at least the reference to Jira

Reply:
removed